litestar/handlers/base.py

```diff
@@ -170,7 +206,7 @@ def fn(self) -> Ref[MaybePartial[AsyncAnyCallable]]:
             Handler function
         """
         if not hasattr(self, "_fn"):
-            raise ImproperlyConfiguredException("Handler has not decorated a function")
+            raise ImproperlyConfiguredException("handler does not decorate a function")
```

Comment:
This error message is ambiguous. 

How about
```suggestion
            raise ImproperlyConfiguredException("No callable has been registered for this handler")
```
?

Reply:
Aight